core/transaction/entity.go

```diff
@@ -269,15 +271,34 @@ func sendTransactionToURL(url string, txn *Transaction, wg *sync.WaitGroup) ([]b
 	return nil, errors.Wrap(err, errors.New("transaction_send_error", postResponse.Body))
 }
 
+type cachedObject struct {
+	Expiration time.Duration
+	Value      interface{}
+}
+
+var cache *lru.Cache
+
+func initLRUCache() {
```

Comment:
use once and package level function `init()`, check how it is done for nonceCache